zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanel.java

```diff
@@ -521,11 +542,19 @@ public void setButtonMode(int mode) {
         private final JToggleButton requestButton;
         private final JToggleButton responseButton;
         private final JToggleButton allButton;
+        private final JToggleButton brkOnJsButton;
```

Comment:
We should be consistent with naming, either `Js` or `JavaScript` (former?).

Reply:
I think that JavaScript Is better. I will rename the one with Js